data.sql

```diff
@@ -0,0 +1,52 @@
+INSERT INTO animals (
+    name
+  , date_of_birth
+  , escape_attempts
+  , neutered
+  , weight_kg
+) VALUES (
+  'Agumon'
+  , 'feb/3/20'
```

Comment:
**Changes Required**
- [ ] This date format is not supported on database level. It simple means you cannot perform comparison queries with them. Please let us instead use supported date format.

Reply:
Thanks. I'll change the format. But my queries were actually working with that form
![Screen Shot 2021-12-01 at 8 01 30](https://user-images.githubusercontent.com/55037686/144249791-a8cf17c5-63da-4505-9a49-2d15c8e615d6.png)
at